DESCRIPTION

```diff
@@ -39,6 +38,8 @@ Imports:
     base64enc,
     shiny,
     rstudioapi
+Remotes:
+    dplyr (>= 0.5.0.9000)
```

Comment:
You need it in both remotes and imports - remotes is just a way to tell devtools where to get it from. R doesn't know anything about it.